src/Sulu/Bundle/AutomationBundle/Tests/Unit/Tasks/Manager/TaskManagerTest.php

```diff
@@ -112,6 +112,10 @@ function (TaskEvent $event) use ($task) {
                     return $task == $event->getTask();
                 }
             )
-        )->willReturnArgument(1);
+        )->will(
```

Comment:
Wouldn't it be better to just add `prophecy` in the version we need in the `require-dev` session in `composer.json`? Then we would have less code to maintain... /cc @chirimoya